src/main/java/hu/dpc/phee/operator/entity/transfer/Transfer.java

```diff
@@ -73,6 +73,10 @@ public class Transfer extends AbstractPersistableCustom<Long> {
     @Column(name = "BATCH_ID")
     private String batchId;
 
+    @Column(name = "ClientCorrelationId")
```

Comment:
why is the formating ( case) not the same as other fields?

Reply:
Will change it